openmdao/core/system.py

```diff
@@ -367,6 +367,10 @@ class System(object):
     _tot_jac : __TotalJacInfo or None
         If a total jacobian is being computed and this is the top level System, this will
         be a reference to the _TotalJacInfo object.
+    _resp_indices_map : dict
```

Comment:
Also, I didn't notice any changes to `Problem.list_problem_vars`.  You might want to make sure the constraints printed there show both the alias and the target.